genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -268,7 +278,9 @@ def _extract_openai_text_output(self, response: str) -> str:
         """Extracts text/content of the OpenAI response object."""
         response = remove_sse_prefix(response)
 
-        if response == "[DONE]":
+        if response == "[DONE]" or not (
+            response.strip().startswith("{") and response.strip().endswith("}")
```

Comment:
I don't know how I feel about this line. This is basically just ignoring any responses that doesn't start or end with proper json format, but given that we already have a lot of custom checks in `_preprocess_response` method, I would rather let the program complain instead of (potentially) silently ignoring it when something's wrong with the response.

Reply:
I had this occur when running against TGI, where everything was formatted correctly excpet for the last chunk with was 1-2 random characters.  I think probably some garbage due to the fact that TGI doesnt send a DONE, (which is now properly killed in GenAI Perf, but seemed to be unhappy)

Reply:
Ah I see so this is to workaround some issues with TGI. In that case, I think it would be better if this PR only handled the fix for sse related error, and maybe create like a follow up ticket to investigate/handle this as a separate work. @debermudez what do you think?